### PR TITLE
Bring Account docs into one section

### PR DESCRIPTION
**Pull request by lornajane**
## Description

This combines the Account API and Secret Management into one section,
with one API reference document.

The API Spec PR is not merged yet https://github.com/Nexmo/api-specification/pull/65 and it's kind of blocked on some rendering issues so this one might be open for a while!

I also have more building blocks to create and add before I'd consider this done but they can follow in a second PR if this one is ready before they are.

### CODE REVIEW

**Comment by mheap** [comment]
I've pushed up some changes that add a link to the API ref / add JSON|XML switching to the OAS rendering.

It feels as though we should use subfolders for the building blocks in this section e.g. instead of `building-blocks/create-a-secret` have `building-blocks/secret-management/create-a-secret`. We've done this for the conversation service and it helps to break things up

**Comment by lornajane** [comment]
Thanks for the extra patches :)

I specifically moved the building blocks to be shared since there will only be 6 of them and I felt they'd be simpler in a short, shared list than to find the correct section first.

**Comment by mheap** [comment]
Ah ok, if there will only be 6 then let's keep them top level.

All of the OAS work blocking this will happen in Jan

File: _documentation/account/guides/secret-management.md [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
-In addition to managing your own API secrets, primary accounts can manage API secrets for any secondary account that it owns. To do this, authenticate with the Nexmo API using your primary account's API key and secret and specify the secondary account's API key as the `account_id` to manage in your [API call](/api/account/secret-management).
-
-## Building Blocks
-
-Building blocks are code snippets showing how to use the API to perform various tasks.
-
-```building_block_list
-product: account/secret-management
-```
+In addition to managing your own API secrets, primary accounts can manage API secrets for any secondary account that it owns. To do this, authenticate with the Nexmo API using your primary account's API key and secret and specify the secondary account's API key as the `account_id` to manage in your [API call](/api/account).
```
Comment (marklewin) [inline _documentation/account/guides/secret-management.md]: You start the sentence with a plural (accounts) and then refer to them in the singular (it).

I think you want to say either:

a primary account can manage the API secret for any secondary account that it owns
OR
primary accounts can manage the API secrets for any secondary accounts that they own

Comment (lornajane) [reply _documentation/account/guides/secret-management.md]: Fixed, thanks!

**Comment by mheap** [comment]
I believe that the work blocking this is now complete and we can pick it up again

**Comment by lornajane** [comment]
Checks failing because I haven't got the OAS merged into the repo and the submodule updated yet ... works OK locally

**Comment by lornajane** [comment]
This would also be complete with a couple of new code snippets, they are almost ready but I'll also wait for the submodules PR (#1537 ) to merge.

**Comment by lornajane** [comment]
Tests are failing because I need to rebase on top of the new curl code snippets submodule change ...

**Pull request merged**